__tests__/app.test.js

```diff
@@ -28,37 +29,45 @@ describe("getTopics", () => {
         .get("/api")
         .expect(200)
         .then(({ body }) => {
-          expect(body).toEqual({
-            "GET /api": {
-              description:
-                "serves up a json representation of all the available endpoints of the api",
-            },
-            "GET /api/topics": {
-              description: "serves an array of all topics",
-              queries: [],
-              exampleResponse: {
-                topics: [{ slug: "football", description: "Footie!" }],
-              },
-            },
-            "GET /api/articles": {
-              description: "serves an array of all articles",
-              queries: ["author", "topic", "sort_by", "order"],
-              exampleResponse: {
-                articles: [
-                  {
-                    title: "Seafood substitutions are increasing",
-                    topic: "cooking",
-                    author: "weegembump",
-                    body: "Text from the article..",
-                    created_at: "2018-05-30T15:59:13.341Z",
-                    votes: 0,
-                    comment_count: 6,
-                  },
-                ],
-              },
-            },
-          });
+          expect(body).toEqual(endpoint);
         });
     });
   });
 });
+
+describe('getArticles', () => {
+    test('200: /api/articles/:article_id', () => {
+        return request(app)
+      .get("/api/articles/1")
+      .expect(200)
+      .then(({ body }) => {
+        body.forEach((column) => {
+          expect(column).toMatchObject({
+            article_id: expect.any(Number),

```

Comment:
As we know what article_id is going to come back, we can be more explicit here and expect it to be 1 👍